docs/architecture/adr-040-storage-and-smt-state-commitments.md

```diff
@@ -54,30 +53,30 @@ We propose to use a KV database for both `SS` and `SC`. The store interface will
 
 State Storage requirements:
 
-+ range queries
-+ quick (key, value) access
-+ creating a snapshot
-+ historical versioning
-+ pruning (garbage collection)
+- range queries
+- quick (key, value) access
+- creating a snapshot
+- historical versioning
+- pruning (garbage collection)
 
 State Commitment requirements:
 
-+ fast updates
-+ tree path should be short
-+ query historical commitment proofs using ICS-23 standard
-+ pruning (garbage collection)
+- fast updates
+- tree path should be short
+- query historical commitment proofs using ICS-23 standard
+- pruning (garbage collection)
 
 ### SMT for State Commitment
 
 A Sparse Merkle tree is based on the idea of a complete Merkle tree of an intractable size. The assumption here is that as the size of the tree is intractable, there would only be a few leaf nodes with valid data blocks relative to the tree size, rendering a sparse tree.
 
 The full specification can be found at [Celestia](https://github.com/celestiaorg/celestia-specs/blob/ec98170398dfc6394423ee79b00b71038879e211/src/specs/data_structures.md#sparse-merkle-tree). In summary:
 
-* The SMT consists of a binary Merkle tree, constructed in the same fashion as described in [Certificate Transparency (RFC-6962)](https://tools.ietf.org/html/rfc6962), but using as the hashing function SHA-2-256 as defined in [FIPS 180-4](https://doi.org/10.6028/NIST.FIPS.180-4).
-* Leaves and internal nodes are hashed differently: the one-byte `0x00` is prepended for leaf nodes while `0x01` is prepended for internal nodes.
-* Default values are given to leaf nodes with empty leaves.
-* While the above rule is sufficient to pre-compute the values of intermediate nodes that are roots of empty subtrees, a further simplification is to extend this default value to all nodes that are roots of empty subtrees. The 32-byte zero is used as the default value. This rule takes precedence over the above one.
-* An internal node that is the root of a subtree that contains exactly one non-empty leaf is replaced by that leaf's leaf node.
+- The SMT consists of a binary Merkle tree, constructed in the same fashion as described in [Certificate Transparency (RFC-6962)](https://tools.ietf.org/html/rfc6962), but using as the hashing function SHA-2-256 as defined in [FIPS 180-4](https://doi.org/10.6028/NIST.FIPS.180-4).
+- Leaves and internal nodes are hashed differently: the one-byte `0x00` is prepended for leaf nodes while `0x01` is prepended for internal nodes.
+- Default values are given to leaf nodes with empty leaves.
```

Comment:
Is it worth mentioning what those default values are (`[]byte{}`)?

Reply:
Yes, will add it.